src/Tests/Sfa.Tl.ResultsAndCertification.IntegrationTests/Repositories/DataExportRepositoryTests/When_GetDataExportCoreResults_IsCalled .cs

```diff
@@ -87,7 +87,7 @@ public async Task Then_Expected_Results_Are_Returned(long aoUkprn, bool hasRecor
                         .Select(x => new CoreResultsExport
                         {
                             Uln = x.TqPathwayAssessment.TqRegistrationPathway.TqRegistrationProfile.UniqueLearnerNumber,
-                            AcademicYear = x.TqPathwayAssessment.TqRegistrationPathway.AcademicYear,
+                            AcademicYear = DbContext.AcademicYear.First(e => e.Year == x.TqPathwayAssessment.TqRegistrationPathway.AcademicYear).Name,
```

Comment:
Should we put first in the variable and check the null check rather than applying directly.